Don't show dirty flag

Since we use the official Linux tarball and patch it with our changes, we have
a dirty Linux by definition. Since this is not a problem, we remove this flag
from the kernel version.

diff --git a/scripts/setlocalversion b/scripts/setlocalversion
index 79f7dd57d..b84e06e20 100755
--- a/scripts/setlocalversion
+++ b/scripts/setlocalversion
@@ -74,9 +74,10 @@ scm_version()
 		fi
 
 		# Check for uncommitted changes
-		if git diff-index --name-only HEAD | grep -qv "^scripts/package"; then
-			printf '%s' -dirty
-		fi
+# WORK: removed to ignore '-dirty' since we patch Kernel on purpose
+#		if git diff-index --name-only HEAD | grep -qv "^scripts/package"; then
+#			printf '%s' -dirty
+#		fi
 
 		# All done with git
 		return
@@ -98,9 +99,10 @@ scm_version()
 
 		# Are there uncommitted changes?
 		# These are represented by + after the changeset id.
-		case "$hgid" in
-			*+|*+\ *) printf '%s' -dirty ;;
-		esac
+# WORK: removed to ignore '-dirty' since we patch Kernel on purpose
+#		case "$hgid" in
+#			*+|*+\ *) printf '%s' -dirty ;;
+#		esac
 
 		# All done with mercurial
 		return
